ls1intum/Artemis
# `Discussion`: Add discussion related features to Artemis user documentation

### Checklist
#### General



### Motivation and Context


We need to add the discussion related feature to the user manual in order to make users aware of them and explain how to make use of them.

### Description

I added a new section to the documentation including all the relevant information for the user as well as explanatory screenshots.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 2 Students
- 1 Programming Exercise with Complaints enabled

1. Check the diff for inconsistencies, bad language, typos etc.
2. Check the resulting, updated documentation to see how the changes would _look like_. You can simply click the according job in the github pipeline to be forwarded to the documentation created out of the current PR version:
<img width="928" alt="Bildschirmfoto 2021-12-01 um 23 52 30" src="https://user-images.githubusercontent.com/41366522/144327401-4b4eb987-1d2e-4deb-880f-1018e9eccee6.png">

### Review Progress





#### Review
